### PR TITLE
Rename access log streams to make it more friendly to posix terminology

### LINKED ISSUE #15708: Use `Stdout` and `Stderr` in canonical proto urls for logging
### description

we recently added protos for logging that use the names `Stderror` and `Stdoutput` - these are derived from the unix names which are actually `stderr` and `stdout`

i think if we dont change these we are going to confuse/annoy people who will not be familiar with this name, and make it harder to remember

if down the road (eg after documenting #15652 ) we decide to change to the common names its going to be much harder to change than addressing the issue now

### PR DESCRIPTION
**Pull request by davinci26**
Commit Message:
Fixes #15708

Risk Level: Low (rename only)
Testing: Just refactor

### CODE REVIEW

**Comment by davinci26** [comment]
waiting to see what the CI thinks of the changes.

File: api/envoy/extensions/access_loggers/stream/v3/stream.proto [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
+    // Configuration to form access log data and format.
+    // If not specified, use :ref:`default format <config_access_log_default_format>`.
+    config.core.v3.SubstitutionFormatString log_format = 1
+        [(validate.rules).message = {required: true}];
+  }
+}
+
+// Custom configuration for an :ref:`AccessLog <envoy_api_msg_config.accesslog.v3.AccessLog>`
+// that writes log entries directly to the operating system's standard error.
+message StdErrAccessLog {
```
Comment (mattklein123) [inline api/envoy/extensions/access_loggers/stream/v3/stream.proto:31]: super nit: inconsistent case between "out" and "Err". Can we be consistent there?

**Review by mattklein123 — CHANGES_REQUESTED** [review]
Thanks for the follow up. I agree this is a big improvement. Just a small comment. cc @phlax to see if there are any other comments. Thank you!

/wait

File: api/envoy/extensions/access_loggers/stream/v3/stream.proto (hunk 1 above)
Comment (davinci26) [reply api/envoy/extensions/access_loggers/stream/v3/stream.proto]: good catch! fixed in fe3bfb8cf431619e7d0dafbf32cb627785c5c740

**Review by mattklein123 — APPROVED** [review]
Thanks LGTM. @phlax any additional requests/comments?

**Review by phlax — APPROVED** [review]
lgtm - with the caveat that im still wondering whether we can just use `Stderr`, `Stdout` and `File` rather than `FileAccessLog` etc

the less we make users type/remember the better

**Comment by davinci26** [comment]
Re: the caveat of renaming.

I thought that we discussed it on the issue and we agreed that name consistency with other loggers is more valuable than shortening the name.

**Comment by phlax** [comment]
> I thought that we discussed it on the issue and we agreed that name consistency with other loggers is more valuable than shortening the name.

the consistency i had in mind was more with `FileAccessLog` and then it occurred to me that this has just been added also

not sure about consistency with other protos tbh

not a blocker - just mho

**Pull request merged**